docs/sources/operations/autoscaling.md

```diff
@@ -0,0 +1,135 @@
+---
+title: Autoscaling in Kubernetes
+weight: 30
+---
+# Autoscaling Grafana Loki in Kubernetes
+
+<!--- TODO: Link to blogpost -->
+
+A Loki cluster will typically handle a workload that varies throughout the day.
+To make Loki easier to operate and optimize the cost of running Loki at scale,
+we have designed a set of resources to help you autoscale your Loki cluster,
+so you don't need to scale up and down resources manually as your workload changes.
+
+## Prerequisites
+
+We use [KEDA (*Kubernetes-based Event Driven Autoscaler*)](https://keda.sh/) to configure autoscaling based
+on Prometheus metrics.  Please refer to their documentation for more information about how to install
+and operate KEDA on your Kubernetes cluster.
+
+## Querier Autoscaling
+
+Queriers pull queries from the query scheduler queue and process them on the querier workers. Therefore, it makes sense to scale based on:
+
+- The scheduler queue size.
+- The queries running in the queriers.
+
+### Scaling metric
+
+The Query Scheduler exposes the `cortex_query_scheduler_inflight_requests` metric.
+It tracks the number of queued queries plus the number of queries currently running in the querier workers.
+The following query is useful to scale queriers based on the inflight requests.
+
+```
```

Comment:
```suggestion
```promql
```

Reply:
I think this may work

Reply:
Unfortunately, that doesn't seem to work 😞 
Neither I could see any example with syntax highlighting.

Reply:
How are you testing it?

Reply:
Move into the `docs` dir and run `make docs`. It creates a local server with the docs. Then go to http://localhost:3002/docs/loki/latest/operations/autoscaling/

Reply:
Hhmm I wonder if that exactly matches the website's Hugo settings; I doubt it?
In any case, let's land this and see what happens. I'm reasonably confident we support promql because of Mimir's docs.

Reply:
Done.